common/src/main/java/apoc/ApocConfig.java

```diff
@@ -73,6 +73,7 @@ public class ApocConfig extends LifecycleAdapter {
     public static final String APOC_CONFIG_JOBS_POOL_NUM_THREADS = "apoc.jobs.pool.num_threads";
     public static final String APOC_CONFIG_JOBS_QUEUE_SIZE = "apoc.jobs.queue.size";
     public static final String APOC_CONFIG_INITIALIZER = "apoc.initializer";
+    public static final String APOC_DECOMPRESSION_LIMIT = "apoc.decompression.limit";
```

Comment:
I would like input from the Neo4j security team on whether having this configurable or hardcode it to 100 is better